openslides_backend/presenter/export_meeting.py

```diff
@@ -41,4 +50,91 @@ def get_result(self) -> Any:
             msg += f" Missing permission: {OrganizationManagementLevel.SUPERADMIN}"
             raise PermissionDenied(msg)
         export_data = export_meeting(self.datastore, self.data["meeting_id"])
+        self.exclude_organization_tags_and_default_meeting_for_committee(export_data)
+        self.add_users(export_data, self.data["meeting_id"])
         return export_data
+
+    def exclude_organization_tags_and_default_meeting_for_committee(
+        self, export_data: Dict[str, Any]
+    ) -> None:
+        self.get_meeting_from_json(export_data)["organization_tag_ids"] = None
+        self.get_meeting_from_json(export_data)[
+            "default_meeting_for_committee_id"
+        ] = None
+
+    def add_users(self, export_data: Dict[str, Any], meeting_id: int) -> None:
+        user_ids = self.get_meeting_from_json(export_data)["user_ids"]
+        if not user_ids:
+            return
+        fields = []
+        for field in User().get_fields():
+            if isinstance(
+                field,
+                (
+                    TemplateCharField,
+                    TemplateHTMLStrictField,
+                    TemplateDecimalField,
+                    TemplateRelationListField,
+                ),
+            ):
+                fields.append(
+                    (
+                        field.get_structured_field_name(meeting_id),
+                        field.get_template_field_name(),
+                    )
+                )
+
+        gmr = GetManyRequest(
+            Collection("user"),
+            user_ids,
+            [
+                "id",
+                "username",
+                "pronoun",
+                "title",
+                "first_name",
+                "last_name",
+                "is_active",
+                "is_physical_person",
+                "password",
+                "default",
+                "can_change_own_password",
+                "gender",
+                "email",
+                "default_number",
+                "default_structure_level",
+                "default_vote_weight",
+                "last_email_send",
+                "is_demo_user",
+                "organization_management_level",
+                "is_present_in_meeting_ids",
+            ]
+            + [field_pair[0] for field_pair in fields],
+        )
+        users: Any = self.datastore.get_many([gmr])[Collection("user")]
+        # remove meta_* keys
+        users = {
+            key: {
+                key_inner: value_inner
+                for key_inner, value_inner in value.items()
+                if not key_inner.startswith("meta_")
+            }
+            for key, value in users.items()
+        }
+
+        for user_key in users:
+            for field_name, field_template_name in fields:
+                if users[user_key].get(field_name):
```

Comment:
use `(field_name := users[user_key].get(field_name)): Syntax and in next line `field_name`

Reply:
I don't like the walroff notation. But I changed it a bit.